lib/logstash/inputs/jdbc.rb

```diff
@@ -162,22 +162,19 @@ def run(queue)
       @scheduler = Rufus::Scheduler.new(:max_work_threads => 1)
       @scheduler.cron @schedule do
         execute_query(queue)
+        update_state_file()
```

Comment:
minor style issue: `update_state_file` vs `update_state_file()`. We skip parentheses for a method call if the method accepts no arguments.